docs/en/observability/apm/security/elastic-stack/access-api-keys.asciidoc

```diff
@@ -57,16 +59,19 @@ In the role descriptors box, assign the appropriate privileges to the new API ke
     }
 }
 ----
-
++
 NOTE: This example only provides privileges for **writing data**.
 See <<apm-feature-roles>> for additional privileges and information.
-
-To set an expiration date for the API key, select **Expire after time**
++
+. To set an expiration date for the API key, select **Expire after time**
 and input the lifetime of the API key in days.
+. Click **Create API key**.
+. {blank}
+include::../../configure/outputs/elasticsearch.asciidoc[tag=configure-beats]
```

Comment:
Including this ends up with a text that includes a link to itself (a recursive link):

<img width="902" alt="image" src="https://github.com/user-attachments/assets/bc1c139f-b65c-4aa7-b445-63682d698dd4" />

Feels a bit weird, but might be ok.